src/components/ButtonGroup/ButtonGroup.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import glamorous from 'glamorous';
+
+import Button from '../Button/Button';
+import constants from '../../constants';
+
+const { borderRadius } = constants;
+
+const ButtonGroupContainer = glamorous.div({
+   display: 'inline-block',
+});
+
+const GroupedButton = glamorous(Button)({
+   borderRadius: 0,
+
+   '&:first-of-type': {
+      borderTopLeftRadius: borderRadius,
+      borderBottomLeftRadius: borderRadius,
+   },
+   '&:last-of-type': {
+      borderTopRightRadius: borderRadius,
+      borderBottomRightRadius: borderRadius,
+   },
+});
+
+const ButtonGroup = ({
+   className,
+   disabled,
+   onMouseEnter,
+   onMouseLeave,
+   children,
+}) => (
+   <ButtonGroupContainer
+      className={className}
+      onMouseEnter={onMouseEnter}
+      onMouseLeave={onMouseLeave}
+   >
+      {React.Children.map(children, child => {
+         if (child.type !== Button) {
+            return child;
+         }
+         return (
+            <GroupedButton
+               disabled={disabled}
+               {...child.props}
+            />
```

Comment:
could this fit in 80 chars? `<GroupedButton disabled={disabled} {...child.props} />`

CI complains here.